pkg/skaffold/gcp/client.go

```diff
@@ -17,60 +17,22 @@ limitations under the License.
 package gcp
 
 import (
-	"context"
-	"sync"
-
-	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/version"
-
-	cstorage "cloud.google.com/go/storage"
-	"github.com/pkg/errors"
-	cloudbuild "google.golang.org/api/cloudbuild/v1"
 	"google.golang.org/api/option"
-)
 
-var (
-	cbclient *cloudbuild.Service
-	cbOnce   sync.Once
-	cbErr    error
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/version"
 )
 
-// CloudBuildClient returns an authenticated client for interacting with
-// the Google Cloud Build API. This client is created once and cached
-// for repeated use in Skaffold.
```

Comment:
So I think we'll now be creating a new client per build?

Reply:
yes. I didn't see the value of caching the client. can only create more problems.